core/server/models/base/index.js

```diff
@@ -913,6 +913,10 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
                 data: data,
                 meta: {pagination: response.pagination}
             };
+        }).catch((err) => {
+            console.log('Error');
```

Comment:
We probably don't want to do `console.log` here?

Reply:
We definitely don't, good 👁 !